doc/deoplete.txt

```diff
@@ -581,6 +583,18 @@ dictionary					*deoplete-source-dictionary*
 		This source collects |g:deoplete#keyword_patterns| keywords
 		from 'dictionary'.
 
+		" Examples:
+		" Sample configuration for dictionary source with multiple
+		" dictionary files.
+		setlocal dictionary+=/usr/share/dict/words
+		setlocal dictionary+=/usr/share/dict/american-english
```

Comment:
Hm... I think it worked even with `set dictionary`. What's wrong with using non-buffer-local set up?